src/Umbraco.Core/Services/ContentServiceExtensions.cs

```diff
@@ -67,20 +68,25 @@ public static IContent CreateContent(this IContentService contentService, string
     public static IEnumerable<string> GetAnchorValuesFromRTEs(this IContentService contentService, int id, string? culture = "*")
     {
         var result = new List<string>();
+
+        //TODO : remove this hack : contentProperty.GetValue(culture) always return null when culture variable is "*"
+        if (culture == "*")
+        {
+            culture = string.Empty;
+        }
```

Comment:
```suggestion
        culture = culture is not "*" ? culture : null;
```
Looking at similar code, it seems like normally it should just default to `null`.